App/Sagas/TextileSagas.ts

```diff
@@ -156,6 +154,31 @@ export function * viewThread ( action: ActionType<typeof UIActions.viewThreadReq
   yield call(NavigationService.navigate, 'ViewThread', { id: action.payload.threadId, name: action.payload.threadName })
 }
 
+export function * getUsername (contact: TextileTypes.Contact) {
```

Comment:
If we end up with any more things like this where we fetch data from endpoints, we should write an API client to clean it up.